logo/logo.py

```diff
@@ -98,7 +98,9 @@ def get_h_line(input_tracker):
         graph_dot_p2.move_to(get_graph_point(input_tracker_p2))
 
         #
-        self.play(ShowCreation(graph),)
+        self.play(
+            ShowCreation(graph),
+        )
```

Comment:
This is black's fault. Could you please remove the trailing comma and run black again? That will fix this. Thanks.